languages/java/exercises/concept/interfaces/src/test/java/RemoteControlCarTest.java

```diff
@@ -0,0 +1,37 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertTrue;
+
+public class RemoteControlCarTest {
+    @Test
+    public void race() {
+        ProductionRemoteControlCar productionCar = new ProductionRemoteControlCar();
+        ExperimentalRemoteControlCar experimentalCar = new ExperimentalRemoteControlCar();
+        TestTrack.race((RemoteControlCar)productionCar);
```

Comment:
to respect track style guidelines, please a whitespace between cast and variable name
example `(RemoteControCar) productionCar`

Reply:
Done